### PR TITLE
Fix Black Status Bar Issue

### PR DESCRIPTION
**Pull request by ashkan18**
# Problem
On Contact Gallery page, status bar is black.
fixes https://github.com/artsy/collector-experience/issues/912

# Solution
Originally tried to fix this by changing `presentViewController:` to a `pushViewController` which fixes the issue, but with this change cancel button doesn't work anymore and if we go with this approach we need to update cancel button in Emission to properly dismiss the view assuming it was pushed.

Better solution, thanks to @orta, was to embed `ARInquiryComponentViewController` in a `ARNAvigationController` and present that instead.

# Selfie :)
screen shot 2018-01-19 at 3 59 53 pm

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">PR is classed as Work in Progress</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_travis">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by orta** [comment]
Wanna pair? I'm afraid switching this from a modal to a push isn't probably going to fly

File: Artsy/View_Controllers/Artwork/ARArtworkViewController+ButtonActions.m [hunk 1]
```diff
@@ -108,7 +108,8 @@ - (void)tappedContactGallery
     }
 
     ARInquiryComponentViewController *inquireVC = [[ARInquiryComponentViewController alloc] initWithArtworkID:self.artwork.artworkID];
-    [[ARTopMenuViewController sharedController] presentViewController:inquireVC animated:YES completion:nil];
+    ARNavigationController* nvc = [[ARNavigationController alloc] initWithRootViewController:inquireVC];
```
Comment (orta) [inline Artsy/View_Controllers/Artwork/ARArtworkViewController+ButtonActions.m]: `ARNavigationController *wrapperNav` please - obj-c uses long words

**Comment by orta** [comment]
rock

**Pull request merged**